src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,35 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MINIMUM_AGE = 18;
+    public static final int MINIMUM_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User can't be null!");
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/218726484-d9eadc3b-ece0-4eeb-a71e-2a8a9ef784e9.png)

Reply:
not fixed

Reply:
<img width="715" alt="image" src="https://user-images.githubusercontent.com/81781479/219876968-51fac572-e088-44b7-9ea7-e7dd7d0fac04.png">